hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/token/impl/entity/AccountImplTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.service.token.impl.entity;
+
+import static com.hedera.node.app.Utils.asHederaKey;
+import static com.hedera.node.app.service.token.entity.Account.HBARS_TO_TINYBARS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.protobuf.ByteString;
+import com.hedera.node.app.spi.key.HederaKey;
+import com.hederahashgraph.api.proto.java.Key;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AccountImplTest {
+    private AccountImpl subject;
+    private final HederaKey key =
+            asHederaKey(
```

Comment:
this formatting looks just wrong

Reply:
I just did spotless. Not sure why itis like that 🤔